timescaledb/overview/release-notes/changes-in-timescaledb-2.md

```diff
@@ -5,15 +5,15 @@ database for time-series data. Driven by user feedback and our experience buildi
 products like [Promscale](https://github.com/timescale/promscale), 2.0 is a major
 milestone and introduces the first multi-node, petabyte-scale relational database
 for time-series. In addition to multi-node capabilities, this release includes new
-features - and improvements to existing ones - focused on giving users more flexibility,
- control over their data and the ability to customize behavior to suit their needs.
+features—and improvements to existing ones—focused on giving users more flexibility,
```

Comment:
```suggestion
features, and improvements to existing ones, focused on giving you more flexibility,
```

em-dashes are technically correct, but I'd prefer to avoid them, especially for parenthetical statements. I would prefer to avoid the parenthesis, or use commas.